lib/slack/channel.js

```diff
@@ -1,15 +1,16 @@
 module.exports = class Channel {
   constructor({
-    id, cache, client, logger,
+    id, workspaceId, cache, client, logger,
   }) {
     this.id = id;
+    this.workspaceId = workspaceId;
     this.cache = cache;
     this.client = client;
     this.logger = logger;
   }
 
   cacheKey(...parts) {
-    return [`channel#${this.id}`].concat(parts).join(':');
+    return [`channel#${this.workspaceId}#${this.id}`].concat(parts).join(':');
```

Comment:
nice. I think the scoping is a great idea. I don't know if it's better, more accurate or consistent to do either:
`ws:#{this.workspaceID}#channel#[...]` or `channel#<wsid>#<channelid>`. Just want to make sure we are doing it this way because it makes sense and not to keep the identifier short.

Reply:
Yeah, I agree it would be even better if we had a `ws:` or something like that as part of this key.

In terms of consistency, I tried to make it similar to the other keys that cache by workspace and channel, examples here: https://github.com/github/ecosystem-primitives/pull/229/files?short_path=0d9829a#diff-0d9829a4a83bd19741db3959bf362d99

And I didn't want to update the structure of all of them to include `ws` 😬